code/modules/uplink/uplink_items/uplink_dangerous.dm

```diff
@@ -23,24 +23,24 @@
 	surplus = 50
 	exclude_modes = list(/datum/game_mode/nuclear/clown_ops)
 
-/datum/uplink_item/dangerous/rawketlawnchair
-	name = "84mm Rocket Propelled Grenade Launcher"
-	desc = "A reusable rocket propelled grenade launcher preloaded with a low-yield 84mm HE round. \
-		Guaranteed to send your target out with a bang or your money back!"
-	item = /obj/item/gun/ballistic/rocketlauncher
-	cost = 8
-	surplus = 30
-	include_modes = list(/datum/game_mode/nuclear)
-
 /datum/uplink_item/dangerous/antitank
 	name = "Anti Tank Pistol"
 	desc = "Essentially amounting to a sniper rifle with no stock and barrel (or indeed, any rifling at all), \
 			this extremely dubious pistol is guaranteed to dislocate your wrists and hit the broad side of a barn! \
 	 		Uses sniper ammo. \
 	 		Bullets tend to veer off-course. We are not responsible for any unintentional damage or injury resulting from inaacuracy."
 	item = /obj/item/gun/ballistic/automatic/pistol/antitank/syndicate
-	cost = 14
-	surplus = 25
+	cost = 20
```

Comment:
@Trilbyspaceclone for the second time leave the pistol alone other than accuracy and shell usage.

Reply:
Outside the scope